### ISSUE TITLE
Login Error! Connection Error!

### ISSUE THREAD

**Issue by MarrenDartin** [andrewsegas/instaboost#8]
instabot@test:~/instaboost/instaboost$ python example.py
03:57 Instabot v1.2.0 started at 16.03.2018 03:57:
03:57 Trying to login as test_account...
03:57 Login error! Connection error!

I am getting this after pulling the latest version. Any idea what could be wrong? Bot was working fine earlier. Thanks.

**Comment by andrewsegas** [comment]
@MarrenDartin 
Yes, this problem is because Instagram now is checking some accounts by phone number,
Try first to login in Instagram web www.instagram.com with your account, after that you'll need to confirm one code number in your phone.
After that you can run the robot like the most happy person in the world :)

**Comment by limerick11** [comment]
@andrewsegas I've been trying for days, but already getting past the security challenge, it still logs Login error! Connection error!
Removing already liked medias..

I have tried clearing the user_agent, but it still not working. After starting the bot, it give me the error, and when I refresh the browser, the security challenge comes up again. This is a never ending loop.

Oddly enough, logging in Safari with my iphone does not throw the security challenge. But when I add the iphone user_agent, it doesn't work.

**Comment by davirami88** [comment]
@andrewsegas Hello, i'm having the same issue, i have clone the repository in an ubuntu virtual machine but when i try to run it always throw  Login error! Connection error!, the thing is that when i run the same example.py file on my windows local machine it runs perfectly. What i want to do is use the ubuntu nohup command to keep running the bot for ever but right now i cannot run it with the normal command python example.py.

**Comment by andrewsegas** [comment]
@davirami88 man, i wrote once about vm, where is your vm running? usually vm's IP are very easy to recognize by instagram or any another social network, knowed vm like amazon, google or others are very difficult to pass the security, you can try to use one different proxy in example.py
